ci-operator/config/openshift/release/openshift-release-master__nightly-4.6.yaml

```diff
@@ -272,8 +272,9 @@ tests:
   steps:
     cluster_profile: gcp-openshift-gce-devel-ci-2
     env:
+      COMPUTE_NODE_TYPE: n1-standard-8
       RT_ENABLED: "true"
-    workflow: openshift-e2e-gcp
+    workflow: openshift-e2e-gcp-realtime
```

Comment:
Looks like `COMPUTE_NODE_TYPE` and `RT_ENABLED` should be set on workflow level and not in test anymore.

Reply:
@petr-muller Do you have some references, I tried to search in the code, and failed to find the definition of ENV variables under the workflow.

Reply:
e.g. https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/aws/builds/openshift-e2e-aws-builds-workflow.yaml#L11-L12

Basically just move the whole `env` from the test to the workflow.